doc/FAQ.rst

```diff
@@ -4,6 +4,117 @@ FAQ
 Here is a list of questions we have either been asked by users or
 potential pitfalls we hope to help users avoid:
 
+**Q: How do I adjust the x/y/z axis bounds (matplotlib's xlim, ylim)?**
+
+**A:** Pass an unpacked dictionary containing the kdims/vdims' names as
+keys and a tuple of the bounds as values into obj.redim.range().
+
+This constrains the bounds of x_col to (0, max(x_col)).
+
+.. code:: python
+
+  curve = hv.Curve(df, 'x_col', 'y_col')
+  curve = curve.redim.range(x_col=(0, None), z_col=(None, 10))
+
+If your column names have spaces, you may predefine a dictionary
+and unpack it in obj.redim.range(). This same method is applicable to
+adjust the range of a color bar.
```

Comment:
Maybe "Q: How do I provide keyword arguments for items with spaces" should be a separate question (perhaps with this example), as this is a very general Python technique not specific to redim.range.